dali/python/backend_impl.cc

```diff
@@ -354,8 +354,9 @@ void ExposeTensorList(py::module &m) { // NOLINT
       R"code(
       List of tensors residing in the GPU memory.
       )code")
-    .def("asCPU", [](TensorList<GPUBackend> &t) -> TensorList<CPUBackend>* {
```

Comment:
I'm a bit afraid of that change. Could you perhaps keep "asCPU" as a deprecated alias?

Reply:
Done